README.md

```diff
@@ -39,34 +119,27 @@ To stop and remove the containers you can use the following command:
 docker compose down
 ```
 
-If you want to run the test-containers use `docker compose -f docker-compose-test.yaml` instead of `docker compose`.
+### 5. Setup keycloak
+
+Go to the keycloak admin console at `/keycloak/admin/master/console/`. \
+There you click on the realm selection on the top-left and click `Create Realm`. \
+Then you upload the `keycloak-realm-template.json` and click Create.
+
+Now the keycloak setup for the project is finished and you can create users at the `Users` tab.
+
+## Advanced Configuration
 
-## setup deployment with automatic updates
+### Run services on different hosts
 
-The containers are updated using watchtower configured in the docker-compose file. To update the docker-compose we use a systemctl timer.
+It is possible to run the servcies on different hosts. To run a service on a different host remove it from the `docker-compose.yaml` and add it to a new docker compose file for the other host (for example `docker-compose-host2.yaml`).
 
-The configuration is written for  a user named `docker-user`. You can create and switch to it with the following lines or use another user and change the configuration.
-```bash
-sudo adduser docker-user --disabled-password --gecos ""
-sudo usermod -aG docker docker-user
-sudo su docker-user
+To make the services working you need to adjust all linkages to the outsourced service and all linkages of the outsourced service. Therefor you also need to make the services accesable from the other host. This can be achieved by publishing the needed port.
+For example if you published the overworld-backend on Port 8080 on host `overworld-host` you change
```

Comment:
```suggestion
To make the services work, you need to adjust all linkages of and to the outsourced service.  
Therefore, you also need to make the services accessible from the other host.  
This can be achieved by publishing the needed port.
For example, if you published the `overworld-backend` on port `8080` on host `overworld-host` you change
```